website/views/options/inventory/mounts.jade

```diff
@@ -6,7 +6,7 @@ mixin mountList(source)
           menu
             each potion in env.Content.hatchingPotions
               - mount = egg.key+"-"+potion.key
-              div(popover-trigger='mouseenter', popover=env.t('mountName', {potion: potion.text(env.language.code), mount: egg.mountText(env.language.code)}), popover-placement='bottom')
```

Comment:
Seems to me like it is needed:

## On Develop

![screen shot 2015-07-18 at 4 50 46 pm](https://cloud.githubusercontent.com/assets/2916945/8763644/376fa99c-2d6d-11e5-9ff8-1df5af50c35d.png)


## On your branch

![screen shot 2015-07-18 at 4 52 07 pm](https://cloud.githubusercontent.com/assets/2916945/8763650/58caf510-2d6d-11e5-95d0-a2d96b9f7dcf.png)